codalab/lib/bundle_cli.py

```diff
@@ -1559,16 +1571,26 @@ def do_run_command(self, args):
 
         targets = self.resolve_key_targets(client, worksheet_uuid, args.target_spec)
         params = {'worksheet': worksheet_uuid}
+
         if args.after_sort_key:
             params['after_sort_key'] = args.after_sort_key
-        new_bundle = client.create(
-            'bundles',
-            self.derive_bundle(RunBundle.BUNDLE_TYPE, args.command, targets, metadata),
-            params=params,
-        )
-
-        print(new_bundle['uuid'], file=self.stdout)
-        self.wait(client, args, new_bundle['uuid'])
+        if args.memoize:
+            dependencies = [key + ':' + bundle_target.bundle_uuid for key, bundle_target in targets]
```

Comment:
Is there a standard function we call on the key and uuid rather than having raw string concatenation?

Reply:
I don't think we currently have a function that calls on the **key and uuid** together. Do you mean to pass the key and uuid **separately** as two parameters to function call?